taxonomy: soy_sauce_vegan

benbenben2 opened the pull request:
### What
added vegan and vegetarian tag for soy sauce.

### Screenshot
![Screenshot_20231223_115932](https://github.com/openfoodfacts/openfoodfacts-server/assets/110821832/39ec05eb-da1b-478d-903b-a99adbc3f033)


### Related issue(s) and discussion
- Fixes #-slack-

aleene commented:
I would not add vegan tags to compound ingredients, as it contains to uncertainties. So it is maybe. It all depends on ingredients. In fact it might contain fish extracts.

benbenben2 commented:
Updated to be maybe as it is a compound ingredient.
+ added filter to skip compound ingredients (any ingredients having "ingredients = [etc.]"

See discussion on Slack
